imagersite/imager_images/forms.py

```diff
@@ -0,0 +1,37 @@
+from django.forms import ModelForm, ImageField
+from .models import Photo, Album
+
+
+class AlbumForm(ModelForm):
+    """
+    Form class for user creating an album.
+    """
+    class Meta:
+        model = Album
+        fields = [
+            'title', 'photos', 'description', 'date_published', 'published'
```

Comment:
Both the forms for the `Album` and `Photo` should not have the `date_published` as a field on the form. Instead you should set the `date_published` based on if the user selected "public" or not.